lib/constants.ts

```diff
@@ -81,13 +81,13 @@ export const EMR_EC2_INSTANCE_ROLE_ARN = '';
 export const EMR_SECURITY_CONFIG_NAME = 'MLSpace-EMR-SecurityConfig';
 
 // Set this to false to disable access logging on all MLSpace S3 buckets and the APIGW
-export const ENABLE_ACCESS_LOGGING = true;
+export const ENABLE_ACCESS_LOGGING = false;
 // If access logs are enabled API Gateway will use the Cloudwatch role for your account
 // if you have an existing role set that ARN here otherwise MLSpace will attempt to create
 // the role for you
 export const APIGATEWAY_CLOUDWATCH_ROLE_ARN = '';
 // Set this to false if you do not want to create an MLSpace specific trail
-export const CREATE_MLSPACE_CLOUDTRAIL_TRAIL = true;
+export const CREATE_MLSPACE_CLOUDTRAIL_TRAIL = false;
```

Comment:
why this off by default?

Reply:
Thanks forgot to revert; will correct in next commit